vaadin-platform-test/src/main/java/com/vaadin/platform/test/ComponentsView.java

```diff
@@ -390,6 +392,13 @@ protected void setPresentationValue(String newPresentationValue) {
         accordion.add("Title", new Paragraph("Content"));
         accordion.setId("accordion");
 
+        CollaborationAvatarGroup collaborationAvatarGroup =
```

Comment:
Is this actually testing the CE? I mean I don't see any binding with an actual field

Reply:
When we attach `CollaborationAvatarGroup`, it updates data in CE to include a new avatar, which will update all the other `CollaborationAvatarGroups` to include that avatar. So, by attaching two avatar groups with the same topic id, and checking that they contain two avatars, we make sure that data syncing with CE works.

We could have also created two fields connected to separate `CollaborationBinders`, and make sure that field values are synced.

Even better would be to open two UIs and make sure that data is synced between those, but that needs lots of extra setup that I wanted to avoid here.

But I was thinking that having one feature that synchronizes data through CE in the same UI would be enough for a smoke test.

Reply:
Gotcha, it's alright then.